### ISSUE TITLE
"Send mail" to support PDF option

### ISSUE THREAD

**Issue by fluzzywuzzy** [mpdf/mpdf#1425]
# Feature request

A client of mine requested that their mpdf generated PDF form should have the "Send email" functionality on a button at the bottom of the page. We got it mostly working the way the client wants, setting up a form with action set to mailto:, and a simple submit button. However, the client needs to have the PDF sent as a PDF in the current state, and it seems it's only possible to send the form data in XFDF format.

There is an option for this in Acrobat for example, I can edit the button generated from MPDF to send a PDF instead, so in my mind it's just passing some setting to the PDF via a MPDF setup, or some value on the form and/or submit button.

If this feature is allready available, please point me in the right direction.

**Comment by finwe** [comment]
I am not sure I understand correctly. You want to somehow send an e-mail from inside the mPDF generated file? Can you point me to a source that describes it/does it?

**Comment by fluzzywuzzy** [comment]
Exactly. PDF's have a "Send email" function, I can trigger it by creating a form in the PDF, and having the action set to "mailto:".
I just want it to send it in another format, it's easy to change the format if I edit the PDF in acrobat, but I can't figure out if it's possible to change at generation from MPDF.

Here's a MPDF generated PDF example that uses this function. The button saying "Skicka mail"
[REGULAR-20210505_1041.pdf](https://github.com/mpdf/mpdf/files/6426502/REGULAR-20210505_1041.pdf)
Try pressing it in Acrobat reader, and it should suggest you to pick a mail-service, etc. for sending it.

The HTML in the file generating this PDF is a bit long, but this function is basically just a form setup like this

`<form action="mailto:" method="post">
    <input type="text" name="input1" value="Fri text">
    <input type="submit" value="Skicka mail" />
</form>`

There's a problem with the submit button-text not being visible in chrome, but ignore that.

The only behaviour I would like to change, is the setting that tells the PDF to send XFDF data. As of right now, everything seems to be working, but I can't tell it to instead send the PDF. Basically this setting in Acrobat, I would like to be able to set in MPDF.
![image](https://user-images.githubusercontent.com/38164002/117130505-6ffa9d80-ada0-11eb-9004-c2c6ca8d375d.png)

Does this make the feature request more clear, or do you need something more/other to make sense of it? Sorry If I'm not the best at explaining it...

**Comment by lems3** [comment]
You can do it in Javascript : 

```twig
<input type="button" name="javascriptButton" value="Javascript Button" onClick="
    this.submitForm({ cURL: 'mailto:email@example.com', cSubmitAs: 'PDF' });
" />
```

It won't work in the Chrome preview, but I think it's a Chrome issue, not a mPDF issue.
